src/test/kotlin/vibroad/service/graphql/VideoSelectionIntegrationTest.kt

```diff
@@ -0,0 +1,37 @@
+package vibroad.service.graphql
+
+import io.kotlintest.specs.StringSpec
+import io.micronaut.test.annotation.MicronautTest
+import strikt.api.expectThat
+import strikt.assertions.all
+import strikt.assertions.isEqualTo
+
+@MicronautTest
+class VideoSelectionIntegrationTest(
+        private val graphQlTestClient: GraphQlTestClient
+) : StringSpec({
+
+    "should return a array containing the single stored video entity" {
+        val result = graphQlTestClient.sendQuery("select_all_videos.txt")
+
+        println(result)
```

Comment:
what is that? :P

Reply:
debugging ... was ist der standard in kotlin / micronaut ? log.info(...) oder println ?

Reply:
https://stackoverflow.com/questions/34416869/idiomatic-way-of-logging-in-kotlin

:D essentially using slf4j with Kotlin-Foo to retrieve a logger